bundle/Controller/Admin/TagController.php

```diff
@@ -772,6 +795,38 @@ public function deleteTagsAction(Request $request, Tag $parentTag = null)
         );
     }
 
+    public function searchTagsAction(Request $request)
+    {
+        $this->denyAccessUnlessGranted('ez:tags:read');
+
+        $tags = null;
+        $searchTerm = null;
+
+        $form = $this->createForm(SearchTagsType::class, null);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $searchTerm = \urlencode($form->getData()['searchTerm']);
+            $page = $request->query->getInt('page', 1);
+            $limit = 10;
```

Comment:
This needs to be configurable.

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249